add notes for n25991 (#557)
bitcoin-core-review-club/website: _posts/2022-09-14-#25991.md

components: ["wallet"]
host: stickies-v
status: upcoming
commit: 10bbb0a
---

## Notes

> - [CoinJoin](https://bitcoinops.org/en/topics/coinjoin) is a trustless protocol for mixing UTXOs from multiple owners in order to make it difficult for outside parties to use the block chain’s transaction history to determine who owns which coin. It involves collaboratively creating a single transaction that spends one or multiple UTXOs from each participant into one or multiple new UTXOs for each participant, breaking the link between inputs and outputs.

Reviewer: Or something similar. The inputs/outputs are still linked since they're from the same transaction, but there's a wider anonymity set.

```suggestion
- [CoinJoin](https://bitcoinops.org/en/topics/coinjoin) is a trustless protocol for mixing UTXOs from multiple owners in order to make it difficult for outside parties to use the block chain’s transaction history to determine who owns which coin. It involves collaboratively creating a single transaction that spends one or multiple UTXOs from each participant into one or multiple new UTXOs for each participant, making it more difficult to trace each output's transaction history.
```

Author: Thanks, updated!
